### DISCUSSION TITLE
how to delete project in beta

### DISCUSSION THREAD

**Discussion by kuldeepsharma1** [community/community#7549] [Projects and Issues] [answered]
how can i delete project in beta project

**Comment by rampd2** [comment]
possible duplicate of #5436
I think it is better to upvote the already existing feedback post instead of creating many.

**Comment by MainakRepositor** [comment]
I have found an answer to this problem.

A **Beta** Project in GitHub cannot be directly deleted but the underlying threads can be deleted and the project can be renamed. 
Let me explain it with the screenshots below.

1. Select a sub-project or thread.
2. Then,
![s](https://user-images.githubusercontent.com/64016811/146129700-caa74b8e-2be5-45ad-a9b1-e697a8154069.png)
3. Click on "Delete from the project" and it is done.


**NB**: Once a  project space is created in Beta means it is created. You cannot delete the project in total but can definitely change its contents. Hope this helps! 😊😇

**Comment by 2percentsilk** [comment, marked as answer]
We have recently shipped the ability to [delete projects](https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need). Give it a try and let us know how it goes.

**Reply by rampd2** [reply]
Works as expected.
Thank you for reporting back.
